### PR TITLE
Fixes #21174 - remove javascript tag from ssh_key form page

### PR DESCRIPTION
**Pull request by amirfefer**
It is more efficient to use html event attribute on element then embeded js script that using jqeury to find that element

### CODE REVIEW

**Review by ohadlevy — APPROVED** [review]
will merge once :green:

**Comment by ohadlevy** [comment]
test failures not related - thanks @amirfefer

**Comment by ares** [comment]
I set the release to 1.17, please don't forget to update redmine issue after merging PRs

**Pull request merged**
